components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.common/src/main/java/org/wso2/carbon/apimgt/micro/gateway/common/dto/OAuthApplicationInfoDTO.java

```diff
@@ -0,0 +1,100 @@
+package org.wso2.carbon.apimgt.micro.gateway.common.dto;
+
```

Comment:
Shall we add Licence headers?

Reply:
We have decided to leave the dtos that were taken from APIM code itself as they are, so it will be easier to maintain.